### PR TITLE
docs(v2 data engine): add V2 Data Engine CPU Mask

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue Longhorn/longhorn#8835

### CODE REVIEW

**Review by jhkrug — CHANGES_REQUESTED** [review]
A couple of tiny suggestions for you to consider. Thanks.

File: content/docs/1.8.0/v2-data-engine/features/configurable-cpu-cores.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+To set CPU cores globally, update the [v2-data-engine-cpu-mask](../../../references/settings#v2-data-engine-cpu-mask) setting using a hexadecimal encoded string. For example:
+
+- Use 0x01 to allocate 1 core
+- Use 0x03 to allocate 2 cores
+- Use 0x07 to allocate 3 cores
+
+## Per-node Configuration
+
+For node-specific CPU core allocation, update the engine's `spec.dataEngineSpec.v2.cpuMask` field with a hexadecimal encoded string. By default, this value is empty, and the v2 data engine will use the global setting specified by `v2-data-engine-cpu-mask`. When a per-node configuration is set, the v2 data engine will prioritize this value over the global setting for that specific node.
```
Comment (innobead) [inline content/docs/1.8.0/v2-data-engine/features/configurable-cpu-cores.md:20]:
```suggestion
For node-specific CPU core allocation, update the `spec.dataEngineSpec.v2.cpuMask` field of the instance manager with a hexadecimal encoded string. By default, this value is empty, and the v2 data engine will use the global setting specified by `v2-data-engine-cpu-mask`. When a per-node configuration is set, the v2 data engine will prioritize this value over the global setting for that specific node.
```

**Review by innobead — CHANGES_REQUESTED** [review]
LGTM, just one comment.

**Review by jhkrug — APPROVED** [review]
lgtm

**Pull request merged**